File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("height: ");
    }
    while(height<0 || height>=8);




    int n = 0;

    for(int x = 0; x<height; x++)
    {
        dot--;
        n++;
        for(int y = 0; y<n; y++)
        {
            for(dot = height-1;dot>0; dot--)
            {
                print(".")
            }
            printf("#");
        }
        printf("\n");

    }

}

File: sentimental-mario-less/mario.py
import cs50



while True:
    height = cs50.get_int("enter a height: ")
    if height > 0 and height < 9:
        break


dot = height
hash = 0

for n in range(height):
    dot -= 1
    n += 1
    for j in range(dot):
        print(".")
    for k in range(hash):
        print("#")
